data/json/recipes/recipe_weapon.json

```diff
@@ -1922,6 +1960,28 @@
     "tools": [  ],
     "components": [ [ [ "glass_shard", 1 ] ], [ [ "rag", 1 ], [ "duct_tape", 10 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "bone_knife",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "skill_required": [ "survival", 1 ],
+    "time": "2 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ "bone", 1 ], "bone_human", 1 ]
+  },
+  {
+    "type": "recipe",
+    "result": "sharp_toothbrush",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "time": "30 sec",
+    "autolearn": true,
+    "components": [ [ "toothbrush_plain", 1 ], [ "hairbrush", 1 ], [ "comb_pocket", 1 ], [ "plastic_chunk", 1 ] ]
```

Comment:
Grinding down with bare hands is gonna be a no

Reply:
I agree, we need a very low-quality grinding quality that can be scattered around ubiquitously.
The interim solution is to have a higher grinding quality, not to have none at all.

Reply:
looking forward to sandpaper